indexer/README.md

```diff
@@ -1,226 +1,176 @@
+Click on link. Your web browser should appear requesting you to login. Login with your UCSC Gmail Account.
+1. Registering the subscription
+   ```
+   hca dss put-subscription --replica <aws or gcp> --callback-url <azul-indexer-url> --es-query '{"query": {"match_all": {}}}'
+   ```
+
+    - `<aws or gcp>`: your preferred storage bucket service
+    - `<azul-indexer-url>`: your azul indexer's url. If you need to know what is your url, following the directions 
+    in the below section *Finding the URL of your Azul Indexer Lambda*.
+1. Copy down the uuid for the subscription.
+
+## Reindexing
+
+Run the command below.
+```
+python test/find-golden-tickets.py --dss-url https://<your DSS URL>c--indexer-url https://<your elasticsearch endpoint> --es-query {}`
+```
+where you replace the `<>` value with your own values.
+- `<your DSS URL>` is the endpoint of the HCA DSS.
+- `<your elasticsearch endpoint>` is url of Azul's Elasticsearch Domain.
+       
+
+## Finding the URL of your Azul Indexer Lambda
+
+1. Click **Services** in the Top Toolbar.
+1. Click **Lambda** under the **Compute** subsection.
+1. Search for the name of your lambda and click name of your lambda.
+1. Click on **API Gateway** in the **Designer** Table. Should bear the very top.
+1. Then, the only table below the **Designer** Table is the **API Gateway** Table. In **API Gateway** Table, the
+click on the caret next to under the gateway name.
+
+After expanding the details, you should see the **Invoke URL**. This is the URL of your Azul Indexer Lambda.
+
+## Methods and Endpoints
 
 |  Methods/Endpoints | Notes |
 | ------------- | ------------- |
 | `<callback_url>`/  | takes in a post request and indexes the bundle found in the request   |
 | es_check() |  returns the ES info, good check to make sure Chalice can talk to ES  |
 
-### Manual Loading
-
-Download and expand import.tgz from Data-Bundle-Examples: https://github.com/HumanCellAtlas/data-bundle-examples/blob/master/import/import.tgz
-Download the test/local-import.py file from this repo. Create an environmental variable `BUNDLE_PATH` that points to the import.tgz files. (Note: There are thousands of files in import.tgz, can specify parts of bundles to download: `import/geo/GSE67835` or `import/geo` or `import`)
-Add environmental variable `ES_ENDPOINT` which points to your ES box or have a localhost running. Optionally, create the name of the ES index to add the files to with the environmental variable `ES_INDEX` (default index is `test-import`)
-Required to have a config.json (like the one in `chalicelib/config.json`)
-
-Run `local-import.py`. Open Kibana to see your files appear. The
-
-Note: Manual loading creates mappings for ES, has some list parsing capability, and if `key` in config.json does not exist, returns a value of "no `key`". (This functionality is not present in the Chalice function yet)
 
 ### Todo List
 
```

Comment:
`how to run find-golden-tickets.py` is now addressed above and can be removed from this todo list.